### PR TITLE
HIG, HKR and HVR mission update

### PR DESCRIPTION
**Pull request by That-Guy-Github1**
Higan will probably need a balance tweek so it can start with it's cores as states either that or a bigger sanzu

### CODE REVIEW

File: events/flavorHKR.txt [hunk 1]
```diff
@@ -532,12 +532,11 @@ country_event = {
...
-				area = Dragon_islands
-				area = Eastern_dragon_islands
-				area = Western_dragon_islands
-			}
-			value = 5
+				region = orbital_region 
+				NOT = { area = crack_of_spacetime }
+				NOT = { area = okazaki_state }
+				NOT = { area = yumemi_space_state }
+			value = 20
```
Comment (Hecatia-Lapislazuli) [inline events/flavorHKR.txt]:
```suggestion
			region = orbital_region 
			NOT = { area = crack_of_spacetime }
			NOT = { area = okazaki_state }
			NOT = { area = yumemi_space_state }
			value = 20
```

File: events/flavorHVR.txt [hunk 2]
```diff
@@ -83,32 +83,130 @@ country_event = {   #Taking over Hell - how to deal with territories that we got
...
 	option = {	### Focus on improving Makai
 		name = flavor_hvr.3.a
-		Makai_area = {
+		makai_peninsula_state = {
 			add_local_autonomy = -25
 			add_province_modifier = {
-				name = hvr_making_heaven_out_of_hell
-				duration = -1
+			name = hvr_making_heaven_out_of_hell
+			duration = -1
```
Comment (Hecatia-Lapislazuli) [inline events/flavorHVR.txt]:
```suggestion
				name = hvr_making_heaven_out_of_hell
				duration = -1
```

File: events/flavorHVR.txt [hunk 3]
```diff
@@ -83,32 +83,130 @@ country_event = {   #Taking over Hell - how to deal with territories that we got
...
-		}	
-		Makai_peninsula_state = {
+		}
+		hokkai_state = {
 			add_local_autonomy = -25
 			add_province_modifier = {
-				name = hvr_making_heaven_out_of_hell
-				duration = -1
+			name = hvr_making_heaven_out_of_hell
+			duration = -1
```
Comment (Hecatia-Lapislazuli) [inline events/flavorHVR.txt]:
```suggestion
				name = hvr_making_heaven_out_of_hell
				duration = -1
```

File: events/flavorHVR.txt [hunk 4]
```diff
@@ -83,32 +83,130 @@ country_event = {   #Taking over Hell - how to deal with territories that we got
...
+		docks_of_makai_state = {
 			add_local_autonomy = -25
 			add_province_modifier = {
-				name = hvr_making_heaven_out_of_hell
-				duration = -1
-			}	
-		}	
-		Makai_coast_state = {
+			name = hvr_making_heaven_out_of_hell
+			duration = -1
```
Comment (Hecatia-Lapislazuli) [inline events/flavorHVR.txt]:
```suggestion
				name = hvr_making_heaven_out_of_hell
				duration = -1
```

File: events/flavorHVR.txt [hunk 5]
```diff
@@ -83,32 +83,130 @@ country_event = {   #Taking over Hell - how to deal with territories that we got
...
+			}
+		}
+		makai_coast_state = {
+			add_local_autonomy = -25
+			add_province_modifier = {
+			name = hvr_making_heaven_out_of_hell
+			duration = -1
 			}
```
Comment (Hecatia-Lapislazuli) [inline events/flavorHVR.txt]:
```suggestion
				name = hvr_making_heaven_out_of_hell
				duration = -1
			}
		}
		obelisk_of_creation_state = {
			add_local_autonomy = -25
			add_province_modifier = {
				name = hvr_making_heaven_out_of_hell
				duration = -1
			}
		}
		rural_makai_esoteria_state = {
			add_local_autonomy = -25
			add_province_modifier = {
				name = hvr_making_heaven_out_of_hell
				duration = -1
			}
		}
		fallen_temple = {
			add_local_autonomy = -25
			add_province_modifier = {
				name = hvr_making_heaven_out_of_hell
				duration = -1
			}
		}
		docks_of_makai_state = {
			add_local_autonomy = -25
			add_province_modifier = {
				name = hvr_making_heaven_out_of_hell
				duration = -1
			}
		}
		pandaemonium = {
			add_local_autonomy = -25
			add_province_modifier = {
				name = hvr_making_heaven_out_of_hell
				duration = -1
			}
		}
		makai_area = {
			add_local_autonomy = -25
			add_province_modifier = {
				name = hvr_making_heaven_out_of_hell
				duration = -1
			}
		}
		creators_chamber_state = {
			add_local_autonomy = -25
			add_province_modifier = {
				name = hvr_making_heaven_out_of_hell
				duration = -1
			}
		}
		makai_fairy_homes_state = {
			add_local_autonomy = -25
			add_province_modifier = {
				name = hvr_making_heaven_out_of_hell
				duration = -1
			}
		}
		western_makai_state = {
			add_local_autonomy = -25
			add_province_modifier = {
				name = hvr_making_heaven_out_of_hell
				duration = -1
			}
		}
		demontown_state = {
			add_local_autonomy = -25
			add_province_modifier = {
				name = hvr_making_heaven_out_of_hell
				duration = -1
			}
		}
		land_of_demons = {
			add_local_autonomy = -25
			add_province_modifier = {
				name = hvr_making_heaven_out_of_hell
				duration = -1
			}
		}
		margatroid_state = {
			add_local_autonomy = -25
			add_province_modifier = {
				name = hvr_making_heaven_out_of_hell
				duration = -1
			}
		}
		makai_coast_state_2 = {
			add_local_autonomy = -25
			add_province_modifier = {
				name = hvr_making_heaven_out_of_hell
				duration = -1
			}
		}
		makai_coast_state = {
			add_local_autonomy = -25
			add_province_modifier = {
				name = hvr_making_heaven_out_of_hell
				duration = -1
			}
```

File: missions/HIG_Missions.txt [hunk 6]
```diff
@@ -276,7 +334,10 @@ hig_internal_slot = {
 		}
 		trigger = {
 			OR = {
+				AND = {
 				NOT = { exists = HVR }
+				owns_core_province = 5032
+				}
```
Comment (Hecatia-Lapislazuli) [inline missions/HIG_Missions.txt]:
```suggestion
				AND = {
					NOT = { exists = HVR }
					owns_core_province = 5032
				}
```

File: missions/HIG_Missions.txt [hunk 7]
```diff
@@ -413,41 +475,32 @@ hig_gensokyo_slot = {
...
 			stability = 2
 			num_of_provinces_owned_or_owned_by_non_sovereign_subjects_with = {
 				OR = {
-					area = Saki_state
-					area = Yachie_state
-					area = Keiki_state
-					area = Great_border
-					area = Western_great_border
+				region = animal_realm_region
+				region = great_border_region
```
Comment (Hecatia-Lapislazuli) [inline missions/HIG_Missions.txt]:
```suggestion
					region = animal_realm_region
					region = great_border_region
```

File: missions/HIG_Missions.txt [hunk 8]
```diff
@@ -413,41 +475,32 @@ hig_gensokyo_slot = {
...
 			}
 			num_of_provinces_owned_or_owned_by_non_sovereign_subjects_with = {
 				OR = {
-					area = Saki_state
-					area = Yachie_state
-					area = Keiki_state
-					area = Great_border
-					area = Western_great_border
+				region = animal_realm_region
+				region = great_border_region
```
Comment (Hecatia-Lapislazuli) [inline missions/HIG_Missions.txt]:
```suggestion
					region = animal_realm_region
					region = great_border_region
```

File: missions/HIG_Missions.txt [hunk 9]
```diff
@@ -517,10 +570,10 @@ hig_gensokyo_slot = {
 		}
 		trigger = {
 			num_of_owned_provinces_with = {
-				region = hell_region
+				region = hell_superregion
```
Comment (Hecatia-Lapislazuli) [inline missions/HIG_Missions.txt]:
```suggestion
				superregion = hell_superregion
```

File: missions/HIG_Missions.txt [hunk 10]
```diff
@@ -695,12 +752,20 @@ hig_reform_gensokyo_slot = {
...
 			4969 = {
 				has_trade_building_trigger = yes
+				OR = {
 				th_has_up_to_date_fort_building = {
 					tag = ROOT
 				}
+				th_has_up_to_date_fort_building = {
+					tag = SNZ
+				}
+				}
```
Comment (Hecatia-Lapislazuli) [inline missions/HIG_Missions.txt]:
```suggestion
				OR = {
					th_has_up_to_date_fort_building = {
						tag = ROOT
					}
					th_has_up_to_date_fort_building = {
						tag = SNZ
					}
				}
```

File: missions/HKR_Missions.txt [hunk 11]
```diff
@@ -1168,10 +1431,7 @@ hakurei_missions_many_visitors_empire_slot = {
 		
 		provinces_to_highlight = {
 			religion = th_shinkism
-			NOT = { area = Makai_area }
-			NOT = { area = Makai_coast_state }
-			NOT = { area = Makai_peninsula_state }
-			NOT = { area = Western_makai_state }
+			region = makai_region
```
Comment (Hecatia-Lapislazuli) [inline missions/HKR_Missions.txt]: Change doesn't seem to make sense from what I recall of her mission tree, what's your reasoning for it?

File: missions/HKR_Missions.txt [hunk 12]
```diff
@@ -1493,21 +1750,37 @@ hakurei_missions_handle_the_curse_expand_shintoism_slot = {
 		position = 6
 
 		provinces_to_highlight = {
-			OR = {
-				area = Mayohiga_state
-				area = Ran_state
-				area = Eastern_mountains
+				OR = {
```
Comment (Hecatia-Lapislazuli) [inline missions/HKR_Missions.txt]:
```suggestion
			OR = {
```

File: missions/HVR_Missions.txt [hunk 13]
```diff
@@ -97,19 +90,141 @@ hvr_slot1_branch = {
...
+					area = lunar_state_56
+					area = lunar_state_57
+					area = lunar_state_58
+					area = lunar_state_59
+					area = lunar_state_60
+					area = lunar_state_61
+					area = lunar_state_62
+					area = lunar_state_63
+					area = lunar_state_64
                    }
```
Comment (Hecatia-Lapislazuli) [inline missions/HVR_Missions.txt]:
```suggestion
				}
```

File: missions/HVR_Missions.txt [hunk 14]
```diff
@@ -602,12 +765,24 @@ hvr_slot2_branch = {
 		position = 3
 		required_missions = { HVR_deal_with_nobles }
 		provinces_to_highlight = { 	
-			area = Yumemi_space_state
+			OR = {
+			area = yumemi_space_state
+			area = crack_of_spacetime
+			area = okazaki_state
+			}
```
Comment (Hecatia-Lapislazuli) [inline missions/HVR_Missions.txt]:
```suggestion
			OR = {
				area = yumemi_space_state
				area = crack_of_spacetime
				area = okazaki_state
			}
```

File: missions/HVR_Missions.txt [hunk 15]
```diff
@@ -707,32 +1039,103 @@ hvr_slot2_branch = {
...
+					owned_by = ROOT	
+					}
+				makai_coast_state_2 = {
+					type = all
+					owned_by = ROOT	
+					}
+				makai_coast_state = {
+					type = all
+					owned_by = ROOT	
+					}
```
Comment (Hecatia-Lapislazuli) [inline missions/HVR_Missions.txt]: Brackets are misaligned

File: missions/HVR_Missions.txt [hunk 16]
```diff
@@ -1190,14 +1722,30 @@ hvr_slot3_branch = {
...
-				value = 4
+				OR = {
+				area = rika_state
+				area = another_sky
+				area = rika_state_3
+				area = panzer_graveyard_state
+				area = pazer_other
+				area = rika_state_2
+				area = panzer_factory_state			
+			}
```
Comment (Hecatia-Lapislazuli) [inline missions/HVR_Missions.txt]:
```suggestion
				OR = {
					area = rika_state
					area = another_sky
					area = rika_state_3
					area = panzer_graveyard_state
					area = pazer_other
					area = rika_state_2
					area = panzer_factory_state			
				}		
```

File: missions/HVR_Missions.txt [hunk 17]
```diff
@@ -1954,7 +2609,7 @@ hvr_slot4_branch = {
 					religion = ROOT
 					owned_by = ROOT
 				}
-				amount = 35
+				amount = 300 # starts at 124
```
Comment (Hecatia-Lapislazuli) [inline missions/HVR_Missions.txt]: Probably slightly less? Maybe 250, so is just doubling starting count.

Might be too easy though if Chireiden or... Sunflower field swap, ig. Or a bit hard if Makai leaves. Not sure.

**Review by Hecatia-Lapislazuli — CHANGES_REQUESTED** [review]

**Review by ComradeFlan — APPROVED** [review]

File: missions/HKR_Missions.txt (hunk 11 above)
Comment (That-Guy-Github1) [reply missions/HKR_Missions.txt]: oops on me misread the initial mission

Comment (That-Guy-Github1) [reply missions/HKR_Missions.txt]: fixed

File: missions/HVR_Missions.txt (hunk 15 above)
Comment (That-Guy-Github1) [reply missions/HVR_Missions.txt]: fixed

File: missions/HVR_Missions.txt (hunk 17 above)
Comment (That-Guy-Github1) [reply missions/HVR_Missions.txt]: reduced to 250 and removed owned by condition, should be easily achievable with a full conquest of Makai by senkai and heaven

**Review by Hecatia-Lapislazuli — APPROVED** [review]

**Pull request merged**